versioned_docs/version-1.3.0-beta.1/how_to_guides/multiarch.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Multiple Architecture Image Support
+sidebar_position: 110
+---
+
+# Multiple Architecture Image/Artifact Support with ORAS
+
+ORAS can help you manage multiple architecture artifacts such as configurations, executables, and container images.
+This example shows how you can use ORAS to manage executables for different operating systems and architectures.
+
+## Pushing the artifacts
+The first step is to push the artifacts to your container registry.
+
+First we push a Linux ARM64 executable:
+```shell
+% oras push --plain-http localhost:15000/oras:linux,arm64  --artifact-platform linux/arm64 --artifact-type 'application/vnd.oci.image.config.v1+json'  bin/linux/arm64/oras:application/octet-stream
```

Comment:
nit: you may use `$` instead `%`.

Reply:
I'd sooner use nothing so copy/paste is easier come to think of it.

Reply:
Changed my mind. The way this document is with command and output, I'll go with `$`.

Done.